### PR TITLE
removed duplicate azure function token provider doc

### PR DESCRIPTION
**Pull request by zhenmichael**
Removing duplicate Token Provider tutorial documentation on fluidframework.com. Users should reference the following DCOM docs for Token Provider implementation guide. 
https://learn.microsoft.com/en-ca/azure/azure-fluid-relay/how-tos/azure-function-token-provider

### CODE REVIEW

**Review by tylerbutler — CHANGES_REQUESTED** [review]
Just removing the file will create a 404 for the old page. Instead, this page should just be updated to EITHER:

1. Have a notice that says, "content has been moved to <new URL>". OR
2. Use Hugo's [aliases](https://gohugo.io/content-management/urls/#example-aliases) feature to make a client-side redirect to the new URL.

**Comment by zhenmichael** [comment]
> Just removing the file will create a 404 for the old page. Instead, this page should just be updated to EITHER:
> 
> 1. Have a notice that says, "content has been moved to ". OR
> 2. Use Hugo's [aliases](https://gohugo.io/content-management/urls/#example-aliases) feature to make a client-side redirect to the new URL.

Updated the document to direct users to learn.microsoft.com similar to https://fluidframework.com/docs/deployment/azure-frs/

File: docs/content/docs/build/tokenproviders.md [hunk 1]
```diff
@@ -4,149 +4,8 @@ menuPosition: 3
 editor: sdeshpande3
 ---
 
-The token provider is responsible for creating and signing tokens that the `@fluidframework/azure-client` uses to make
-requests to the Azure Fluid Relay. There are two ways by which you can generate the token:
+In the [Fluid Framework](https://fluidframework.com/), TokenProviders are responsible for creating and signing tokens that the `@fluidframework/azure-client` uses to make requests to the Azure Fluid Relay service. The Fluid Framework provides a simple, insecure TokenProvider for development purposes, aptly named **InsecureTokenProvider**. Each Fluid service must implement a custom TokenProvider based on the particular service's authentication and security considerations.
```
Comment (tylerbutler) [inline docs/content/docs/build/tokenproviders.md:7]:
```suggestion
In the [Fluid Framework](https://fluidframework.com/), TokenProviders are responsible for creating and signing tokens that the `@fluidframework/azure-client` uses to make requests to the Azure Fluid Relay service. Each Fluid service must implement a custom TokenProvider based on the particular service's authentication and security considerations.
```

File: docs/content/docs/build/tokenproviders.md [hunk 2]
```diff
@@ -4,149 +4,8 @@ menuPosition: 3
...
 
-- `InsecureTokenProvider`
-- Implement your own `TokenProvider` class for fetching the token and a backend for token generation
+Each Azure Fluid Relay resource you create is assigned a **tenant ID** and its own unique **tenant secret key**. The secret key is a **shared secret**. Your app/service knows it, and the Azure Fluid Relay service knows it. TokenProviders must know the secret key to sign requests, but the secret key can't be included in client code.
```
Comment (tylerbutler) [inline docs/content/docs/build/tokenproviders.md:10]:
```suggestion
```

I say just remove this.

**Review by tylerbutler — APPROVED** [review]

File: docs/content/docs/build/tokenproviders.md [hunk 3]
```diff
@@ -4,149 +4,8 @@ menuPosition: 3
...
-
-export default httpTrigger;
-```
-
-The `generateToken` function will generate a token for the given user that is signed using the tenant's secret key. This allows the token to be returned to the client without ever exposing the secret itself to it. Instead, the token is generated using it to provide scoped access to the given document. This token can be returned by an ITokenProvider implementation to use with the AzureClient.
-
-## Adding custom data to tokens
-
-You can add custom data such as email id, gender, address, etc for your token generation. See `Token Provider` under [Connect to Azure Fluid Relay]({{< relref "azure-frs.md" >}}) for more information.
+To learn more about using TokenProviders, see [How to: Write a TokenProvider with an Azure Function](https://learn.microsoft.com/en-us/azure/azure-fluid-relay/how-tos/azure-function-token-provider).
```
Comment (tylerbutler) [inline docs/content/docs/build/tokenproviders.md:11]:
```suggestion
To learn more about using TokenProviders, see [How to: Write a TokenProvider with an Azure Function](https://learn.microsoft.com/azure/azure-fluid-relay/how-tos/azure-function-token-provider).
```

If you remove the en-us from the URL, it'll redirect to localized docs if available.

**Pull request merged**